docs/installation/add-to-a-website.md

```diff
@@ -74,26 +79,279 @@ In order to do this, you'll need to build your website with a JavaScript bundler
 like Webpack or Parcel. The rest of the installation methods on this page
 describe how to get started using these tools.
 
-## create-react-app
+## Install with a Package Manager
 
-To add React Router to a
-[create-react-app](https://github.com/facebook/create-react-app) project, you
-can use [yarn](https://yarnpkg.com).
+Before using a bundler for your project, you'll first need to install React
+Router to your local `node_modules` directory using a JavaScript package
+manager. The following instructions use [npm](https://www.npmjs.com/), but
+[Yarn](https://yarnpkg.com/) is also a popular choice.
 
+```sh
+$ npm install history@5 react-router@6 react-router-dom@6
 ```
-$ yarn add react-router@6 react-router-dom@6
+
+## Create React App
+
+Follow the instructions in the [React documentation to set up a new project with Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app), then follow [the instructions above](/#install-with-a-package-manager)
+to install React Router in your project.
+
+Once your project is set up and React Router is installed as a dependency, open
+the `src/index.js` in your text editor. Import `BrowserRouter` from
+`react-router-dom` near the top of your file.
+
+```diff
+import React from 'react';
+import ReactDOM from 'react-dom';
++ import { BrowserRouter as Router } from "react-router-dom";
```

Comment:
Nice, love how you show the diff here @chancestrickland. Makes it super easy to follow 👍